test/karma.conf.js

```diff
@@ -1,7 +1,7 @@
 /*eslint no-var:0, object-shorthand:0 */
 
 var coverage = String(process.env.COVERAGE)!=='false',
-	sauceLabs = String(process.env.SAUCELABS).match(/^(1|true)$/gi) && !String(process.env.TRAVIS_PULL_REQUEST).match(/^(1|true)$/gi),
+	sauceLabs = String(process.env.SAUCELABS).match(/^(1|true)$/gi) && String(process.env.TRAVIS_PULL_REQUEST).match(/^(0|false)$/gi),
```

Comment:
Might be worth also allowing TRAVIS_PULL_REQUEST to be undefined here? (`/^(0|false|undefined)$/gi`)